frontend/src/components/viewers/MarkdownViewer.tsx

```diff
@@ -75,10 +79,41 @@ class MarkdownViewer extends Viewer<MarkdownViewerProps, any> {
     }
     return (
       <div className='markdown-viewer'>
-        <Markdown>{this._config.markdownContent}</Markdown>
+        <MarkdownAdvanced
+          maxMarkdownStrLength={
+            this.props.maxMarkdownStrLength ? this.props.maxMarkdownStrLength : undefined
```

Comment:
nit: do we need the ? expression? Looks like we can directly pass maxMarkdownStrLength

Reply:
That is a good point! Fixed.